lib/bolt/outputter/rainbow.rb

```diff
@@ -8,6 +8,12 @@ class Rainbow < Bolt::Outputter::Human
       def initialize(color, verbose, trace, stream = $stdout)
         begin
           require 'paint'
+          if Bolt::Util.windows?
```

Comment:
This just tells us that we're on Windows. To *really* tell if this will work, we'll have to detect we're on Windows 10, which since we're bolt a call to facter is relatively easy. Do we want the potential perf hit or do we want to allow it to work on any Windows and document that it will only display correctly on Windows 10.